### DISCUSSION TITLE
ModuleNotFoundError: No module named 'thinc.neural'

### DISCUSSION THREAD

**Discussion by atesas** [explosion/spaCy#8936] [Help: Installation] [answered]
Hi guys! 

When I try loading en_core_web_trf model (or any others) I get the following error.

```
    from thinc.neural.util import get_array_module
ModuleNotFoundError: No module named 'thinc.neural'
```

My code has been working well in the last months and this is the first time I am receiving this error! Is it related to updates? (Also I started using prodigy, do you think the error has to do with some installations that I made for prodigy?)

Any help is appreciated!

PS: I tried re-installing spacy and thinc and it didn't work!

**Comment by polm** [comment, marked as answer]
Can you give the output of `spacy info` and `pip freeze`? It's hard to figure out what's happening without knowing your versions/OS.
